internal/backend/db/dbgorm/scheme/scheme.go

```diff
@@ -76,6 +76,8 @@ type Connection struct {
 	StatusCode    int32 `gorm:"index"`
 	StatusMessage string
 
+	DeletedAt gorm.DeletedAt `gorm:"index"`
```

Comment:
Don't you need to update queries accordingly to not fetch deleted?

Reply:
gorm is handling this

Reply:
there is this fixture - assertDeleted, which tests exactly this - tests that find query will skip deleted objects and raw db access will still find them